spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/Span.java

```diff
@@ -127,4 +130,26 @@
 	 * Will never be null.
 	 */
 	List<Log> logs();
+
+
+	/**
+	 * Class used for conversions of long ids to their String representation
+	 */
+	class Converter {
```

Comment:
Style feedback

this reads `Span.Converter.toHexString(id)` I think this distracts the span type a bit, as it only applies to its id. Ex converter (or codec) makes me think about the span, and then I get to hexString and notice it is looking for an id.

ps Since hex is already a string, I don't think you need to qualify the methods by appending String to them.

I think SpanId.toHex/fromHex or even Hex.encode/decode would be better. Use `SpanId` or similar, if you want to clarify the relationship about ids. Use `Hex.` if you are just looking to have these codecs somewhere common. Both are valid, imho, but in the end.. I'd probably choose Hex.encode/decode as it is shorter.